build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -820,20 +1316,32 @@ new file mode 100644
 +    return;
 +  scheduled_ = true;
 +
++  ReStart(initial_check_delay);
++}
++
++void AdBlockUpdaterService::ReStart(int delay) {
++  int days = GetAdBlockUpdateTimePeriod();
++  if (days <= 0) days = 7;
++  next_check_delay = 60*60*24*days;
++
++  if (delay == 0) delay = next_check_delay;
++
 +  LOG(INFO) << "AdBlockUpdaterService: starting up. "
 +          << "First update attempt will take place in "
-+          << initial_check_delay << " seconds. "
++          << delay << " seconds. "
 +          << "Next update attempt will take place in "
 +          << next_check_delay << " seconds. ";
 +
++  scheduler_->Stop();
 +  scheduler_->Schedule(
-+      base::TimeDelta::FromSeconds(initial_check_delay),
++      base::TimeDelta::FromSeconds(delay),
 +      base::TimeDelta::FromSeconds(next_check_delay),
 +      base::Bind(&AdBlockUpdaterService::OnDemandScheduledUpdate,
 +                 base::Unretained(this)), base::DoNothing());
 +}
 +
-+void AdBlockUpdaterService::OnDemandScheduledUpdate(component_updater::UpdateScheduler::OnFinishedCallback on_finished) {
++void AdBlockUpdaterService::OnDemandScheduledUpdate(
++        component_updater::UpdateScheduler::OnFinishedCallback on_finished) {
 +  //TODO: call on_finished
```

Comment:
Is this a new TODO?

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/b11ab2588e787d6349664120c0ccdb7827cab201

Reply:
It is fixed because now `on_finished` is called correctly?

Reply:
yes, the old patch didn't use it, now it's called up.
however, the called function is currently `base::DoNothing()`

https://source.chromium.org/chromium/chromium/src/+/master:components/component_updater/timer_update_scheduler.cc;l=21;drc=8cfbfc3a40cb9513286d6a94cbf17783d558745b;bpv=1;bpt=1